### DISCUSSION TITLE
是否考虑接入「阿里云盘」这个存储服务的支持

### DISCUSSION THREAD

**Discussion by Coding-Yuan** [remotely-save/remotely-save#404] [Ideas]
阿里云盘现在有开放平台的 API 可以接入了，作者是否考虑将其加入到支持的存储服务列表中 🙏

**Comment by exoticknight** [comment]
麻烦你可以贴上一些资料供我们去了解研究一下？

**Comment by fyears** [comment]
这里最大的问题是 api 不知道怎么申请以及申请需要什么条件

**Comment by fyears** [comment]
找到了对应接入文档：

https://www.yuque.com/aliyundrive/zpfszx/btw0tw

最大的问题是，需要 提供备案和实名认证

**Reply by 161424** [reply]
我觉得可以通过alist，在阿里云备份。但是alist要是公网访问，还需要进行打洞，呃呃
